CHANGELOG.md

```diff
@@ -76,6 +76,7 @@
 * [BUGFIX] Configs: prevent validation of templates to fail when using template functions. #3157
 * [BUGFIX] Configuring the S3 URL with an `@` but without username and password doesn't enable the AWS static credentials anymore. #3170
 * [BUGFIX] Limit errors on ranged queries (`api/v1/query_range`) no longer return a status code `500` but `422` instead. #3167
+* [BUGFIX] No-longer-needed ingester operations from queries are now canceled. #3178
```

Comment:
Typo in `queriers`.

Reply:
I left the same comment, but Bryan correctly told it's intended to be "queries" because it affects both queriers and rulers.

Reply:
Well, it confused two of us, so perhaps better wording like "operations caused by queries" would improve it?